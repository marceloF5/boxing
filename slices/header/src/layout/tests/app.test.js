import React from 'react';
import { mount } from 'enzyme';
import App from '../app';

describe('App', () => {
    test('should render', () => {
        const wrapper = mount(<App />);

        expect(wrapper.exists()).toBeTruthy();
    });
});
