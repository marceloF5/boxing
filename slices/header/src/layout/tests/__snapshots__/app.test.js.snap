// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
"<div>
  <h1>
    Header
  </h1>
</div>"
`;
